src/components/Routes/Routes.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Navigate } from 'react-router';
+import { About } from '../About';
+import { Agencies } from '../Agencies';
+import { Astronauts } from '../Astronauts';
+import { Events } from '../Events';
+import { Home } from '../Home';
+import {
+  homeUrl,
+  eventsUrl,
+  agenciesUrl,
+  astronautsUrl,
+  aboutUrl,
+} from './routesUrl';
+
+export const rootRoutes = [
+  {
+    path: homeUrl,
+    element: <Home />,
+  },
+  {
+    path: eventsUrl,
+    element: <Events />,
+  },
+  {
+    path: agenciesUrl,
+    element: <Agencies />,
+  },
+  {
+    path: astronautsUrl,
+    element: <Astronauts />,
+  },
+  {
+    path: aboutUrl,
+    element: <About />,
+  },
+  {
+    path: '*',
+    element: <Navigate to={homeUrl} replace={true} />,
```

Comment:
For what purposes `replace` is used?

Reply:
a call to navigate will push a new entry into the history stack so the user can click the back button to get back to the page. If you pass replace: true to navigate then the current entry in the history stack will be replaced with the new one.

Reply:
a call to navigate will push a new entry into the history stack so the user can click the back button to get back to the page. If you pass replace: true to navigate then the current entry in the history stack will be replaced with the new one.

Reply:
It was just a question. Did not need to be removed

Reply:
as I understand  it is not necessary in this case